test_scripts/RC/Capabilities/007_resend_only_supported_parameters.lua

```diff
@@ -0,0 +1,62 @@
+---------------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [SDL_RC] Set available control module settings SetInteriorVehicleData
+--
+-- Description:
+-- In case:
+-- 1) SDL receive several supported Radio parameters in GetCapabilites response
+-- SDL must:
+-- 1) Transfer to HMI rempote control RPCs only with supported parameters and
+-- 2) Reject any request for RADIO with unsupported parameters with UNSUPPORTED_RESOURCE result code, success: false
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonRC = require('test_scripts/RC/commonRC')
+
+--[[ Local Variables ]]
+local radio_capabilities = {moduleName = "Radio", radioFrequencyAvailable = true, radioBandAvailable = true}
+
+local rc_capabilities = commonRC.buildHmiRcCapabilities(commonRC.DEFAULT, radio_capabilities, commonRC.DEFAULT)
+local available_params =
+{
+    moduleType = "RADIO",
+    radioControlData = {frequencyInteger = 1, frequencyFraction = 2, band = "AM"}
+}
+local absent_params = {moduleType = "RADIO", radioControlData = {frequencyInteger = 1, frequencyFraction = 2}}
+
+--[[ Local Functions ]]
+local function setVehicleData(params, self)
+	local cid = self.mobileSession:SendRPC("SetInteriorVehicleData", {moduleData = params})
+
+	if params.radioControlData.frequencyInteger then
+		EXPECT_HMICALL("RC.SetInteriorVehicleData",	{
+            appID = self.applications["Test Application"],
+			moduleData = params})
+		:Do(function(_, data)
+				self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {
+					moduleData = params})
+			end)
+		self.mobileSession:ExpectResponse(cid, { success = true, resultCode = "SUCCESS" })
+	else
+		EXPECT_HMICALL("RC.SetInteriorVehicleData"):Times(0)
+		self.mobileSession:ExpectResponse(cid, { success = false, resultCode = "UNSUPPORTED_RESOURCE" })
+	end
+end
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", commonRC.preconditions)
+runner.Step("Start SDL, HMI, connect Mobile, start Session", commonRC.start, {rc_capabilities})
+runner.Step("RAI, PTU", commonRC.rai_ptu)
+runner.Step("Activate_App", commonRC.activate_app)
+
+runner.Title("Test")
+for _,module_name in pairs({"CLIMATE", "RADIO"}) do
```

Comment:
@Itileda Please add space after comma:
`for _,module_name in pairs` -> `for _, module_name in pairs`